corehq/apps/reports/v2/reports/explore_case_data.py

```diff
@@ -51,12 +55,22 @@ class ExploreCaseDataReport(BaseReport):
         DateXpathColumnFilter,
     ]
 
+    report_filters = [
+        CaseOwnerReportFilter,
+    ]
+
     def _get_base_query(self):
-        return CaseSearchES().domain(self.domain)
+        return (CaseSearchES()
+                .domain(self.domain)
+                .NOT(case_es.case_type("user-owner-mapping-case")))
```

Comment:
Can you use the constant `USER_LOCATION_OWNER_MAP_TYPE` here? Should this do anything with other special case types, like `supply-point`? I don't know much about this code, I'm just looking at [commtrack/const.py](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/commtrack/const.py).